llvm/lib/Target/AMDGPU/AMDGPUISelLowering.cpp

```diff
@@ -316,6 +316,7 @@ AMDGPUTargetLowering::AMDGPUTargetLowering(const TargetMachine &TM,
   setTruncStoreAction(MVT::v2f32, MVT::v2f16, Expand);
   setTruncStoreAction(MVT::v3f32, MVT::v3f16, Expand);
   setTruncStoreAction(MVT::v4f32, MVT::v4f16, Expand);
+  setTruncStoreAction(MVT::v4f32, MVT::v4bf16, Expand);
```

Comment:
Can you also look at other vector size like v2bf16, v8bf16, etc for TruncStoreAction? Thanks.

Reply:
Yeah, I was about to ask if we want to add them all. Okay will do.